src/algokit/cli/tasks/send_transaction.py

```diff
@@ -0,0 +1,166 @@
+import json
+import logging
+from io import TextIOWrapper
+from pathlib import Path
+from typing import cast
+
+import click
+from algosdk import encoding, error
+from algosdk.transaction import SignedTransaction, retrieve_from_file
+
+from algokit.cli.common.utils import MutuallyExclusiveOption
+from algokit.cli.tasks.utils import get_transaction_explorer_url, load_algod_client, stdin_has_content
+
+logger = logging.getLogger(__name__)
+
+
+def _is_sign_task_output_txn(item: dict) -> bool:
+    """
+    Checks if a given item is a dictionary and contains the keys "txn_id" and "content".
+
+    Args:
+        item (dict): A dictionary object to be checked.
+
+    Returns:
+        bool: True if the input item is a dictionary with the keys "txn_id" and "content", False otherwise.
+    """
+
+    return isinstance(item, dict) and all(key in item for key in ["txn_id", "content"])
+
+
+def _load_from_stdin() -> list[SignedTransaction]:
+    """
+    Load transaction data from standard input and convert it into a list of SignedTransaction objects.
+
+    Returns:
+        A list of SignedTransaction objects representing the loaded transactions from the standard input.
+
+    Raises:
+        click.ClickException: If the piped transaction content is invalid.
+    """
+    # Read the raw file content from the standard input
+
+    raw_file_content = cast(TextIOWrapper, click.get_text_stream("stdin")).read()
+
+    try:
+        # Parse the raw file content as JSON
+        file_content = json.loads(raw_file_content)
+    except json.JSONDecodeError as ex:
+        raise click.ClickException("Invalid piped transaction content!") from ex
+
+    # Check if the content is a list of dicts with the required fields
+    if not isinstance(file_content, list) or not all(_is_sign_task_output_txn(item) for item in file_content):
+        raise click.ClickException("Invalid piped transaction content!")
+
+    # Convert the content into SignedTransaction objects
+    return [encoding.msgpack_decode(item["content"]) for item in file_content]  # type: ignore[no-untyped-call]
```

Comment:
what is `"content"`?

Reply:
Should we use `transaction` instead to be clearer? What format is it in?

Reply:
@robdmoore renamed into `transaction_id` its just a string value.